frontend/src/libraries/explorer-wamp/accounts.ts

```diff
@@ -4,6 +4,7 @@ export interface AccountBasicInfo {
   id: string;
   timestamp: number | string;
   address: string;
+  blockHash?: string;
```

Comment:
What is a block hash of an account?

Well, I believe, the best solution here is to make the `storagePaidAt` block height conversion to block hash right in the Account Details component just using the component state for that. To avoid blocking on waiting for the block hash, you can render without a `<BlockLink>` first (just render as text `#100500`) and once we resolve the `storagePaidAtBlockHash`, we can render that cell with `<BlockLink>`.

Thus, this change should be reverted.